tests/api.c

```diff
@@ -21046,6 +21284,77 @@ static void test_wc_SetIssuerRaw(void)
 #endif
 }
 
+static void test_wc_SetIssueBuffer(void)
+{
+#if !defined(NO_ASN) && !defined(NO_FILESYSTEM) && defined(OPENSSL_EXTRA) && \
+    defined(WOLFSSL_CERT_GEN) && defined(WOLFSSL_CERT_EXT)
+    char joiCertFile[] = "./certs/test/cert-ext-joi.pem";
```

Comment:
Can this be a `const char* = `? Looks like several more below that can be updated as well. Some compilers will not like having just `char` here since the "" string is const.